### DISCUSSION TITLE
Whats the use of emailVerified property in database

### DISCUSSION THREAD

**Discussion by SahilMahadwar** [nextauthjs/next-auth#3360] [Help] [answered]
Hey I was just wondering whats the use of emailVerified  property its always null I used 2 providers GitHub and google its still shows nul
![Screenshot 2021-12-04 234905](https://user-images.githubusercontent.com/37921525/144720304-78630b6f-f0a3-46a7-8dc7-c26070524f33.jpg)
l

**Comment by balazsorban44** [comment, marked as answer]
Using the email provider https://next-auth.js.org/providers/email you can verify if an email belongs to a user.
